.travis.yml

```diff
@@ -31,11 +31,35 @@ matrix:
         - os: osx
           language: generic
           env: TERRYFY_PYTHON='macpython 3.6.0'
+        - services:
+            - docker
+          env: DOCKER_IMAGE=quay.io/pypa/manylinux1_x86_64
+          before_install:
+            - if ! [[ $TRAVIS_TAG ]]; then exit 0; fi
+          install:
+            - docker pull $DOCKER_IMAGE
+          script:
+            - docker run --rm -v `pwd`:/io $DOCKER_IMAGE /io/.manylinux-install.sh
+          after_success:
+            - pip install twine && twine upload -u zope.wheelbuilder -p $PYPIPASSWORD wheelhouse/*
+            - exit 0
+        - services:
+            - docker
+          env: DOCKER_IMAGE=quay.io/pypa/manylinux1_i686
+          before_install:
+            - if ! [[ $TRAVIS_TAG ]]; then exit 0; fi
+          install:
+            - docker pull $DOCKER_IMAGE
+          script:
+            - docker run --rm -v `pwd`:/io $DOCKER_IMAGE /io/.manylinux-install.sh
+          after_success:
+            - pip install twine && twine upload -u zope.wheelbuilder -p $PYPIPASSWORD wheelhouse/*
+            - exit 0
```

Comment:
I had no idea Travis matrices allow you to override everything, including the build script!  TIL.